api_reference.md

```diff
@@ -31,8 +31,8 @@ The maximum text size per request is 3000 bytes.
     comment. A comment could be a single post to a web page's comments section,
     a forum post, a message to a mailing list, a chat message, etc.
 
-*   A **model** is a dimension that the comment is scored on, for example,
-    "toxic", "obscene", "thoughtful", "off-topic", etc. This is also sometimes referred to
+*   A **model** is a dimension that the comment is scored on; for example,
+    "toxic," "obscene," "thoughtful," "off-topic," etc. This is also sometimes referred to
```

Comment:
I'm not sure I'm on board with this prescriptivist motivated change, but not blocking on it.

Reply:
Hah. I went back and forth on this one, actually. Most major style guides would agree with me on this, at least grammatically. But it's a bit of a hybrid situation, since these dimensions, quotes included, also count as data.